applications/AdjointFluidApplication/custom_utilities/numerical_diffusion.h

```diff
@@ -0,0 +1,407 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 license: KratosAdjointFluidApplication/license.txt
+//
+//  Main author:    Suneth Warnakulasriya, https://github.com/sunethwarna
+//
+
+
+#if !defined(KRATOS_NUMERICAL_DIFFUSION)
+#define KRATOS_NUMERICAL_DIFFUSION
+
+// System includes
+#include <vector>
+#include <string>
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+#include "includes/process_info.h"
```

Comment:
not needed if you include modelpart, (which btw is included twice now)

Reply:
done